src/Files.App/Utils/Storage/Operations/FilesystemHelpers.cs

```diff
@@ -869,7 +870,9 @@ public static bool ContainsRestrictedFileName(string input)
 		public void Dispose()
 		{
 			filesystemOperations?.Dispose();
-
+   
+	 // SUPPRESS: Cannot convert null literal to non-nullable reference type.
+	#pragma warning disable CS8625
 			associatedInstance = null;
 			filesystemOperations = null;
```

Comment:
They are assigned values in the constructor and shouldn't ever be null, and are only assigned null when the object is Disposed, so it won't be in use anymore from what I understand. So the values are only null when the object is no longer used, so in this case it made sense to me to supress the warning since the compiler doesn't have that context. If the two are marked as nullable then that will probably cascade into many more warning in the file, even though those fields shouldn't be null. If you think that this isn't a good solution, we can come up with something better

Reply:
Okay, then I don't think the null assignment is necessary, since references to the disposed object should be immediately lost, so references to these objects should also be lost without explicitly nulling them here.

Reply:
The documentation on how Dispose works in C# is a bit weighty, so I am not sure if this is the most ideal implementation. However, from what I understand, the assignment of null signals to the Garbage Collector that the object is safe to remove from memory and seems to be standard practice in the application. This may not be the case but I haven't done enought research to be certain enough be make that change.

Reply:
I still think the null assignment is unnecessary, but the purpose of this PR is to remove the warnings, so if you think it's better to leave it as is, then leave it as is.